apps/web/src/pages/EmployeeProfile/EmployeeProfilePage.tsx

```diff
@@ -0,0 +1,219 @@
+import { useIntl } from "react-intl";
+import { useQuery } from "urql";
+import ChartBarSquareIcon from "@heroicons/react/24/outline/ChartBarSquareIcon";
+
+import { navigationMessages } from "@gc-digital-talent/i18n";
+import { FragmentType, getFragment, graphql } from "@gc-digital-talent/graphql";
+import {
+  Heading,
+  Pending,
+  TableOfContents,
+  ThrowNotFound,
+} from "@gc-digital-talent/ui";
+import { ROLE_NAME } from "@gc-digital-talent/auth";
+import { UnauthorizedError } from "@gc-digital-talent/helpers";
+
+import Hero from "~/components/Hero";
+import SEO from "~/components/SEO/SEO";
+import useBreadcrumbs from "~/hooks/useBreadcrumbs";
+import useRoutes from "~/hooks/useRoutes";
+import RequireAuth from "~/components/RequireAuth/RequireAuth";
+import { isVerifiedGovEmployee } from "~/utils/userUtils";
+import profileMessages from "~/messages/profileMessages";
+import StatusItem from "~/components/StatusItem/StatusItem";
+
+import messages from "./messages";
+
+const SECTION_ID = {
+  CAREER_PLANNING: "career-planning-section",
+  TALENT_MOBILITY: "talent-mobility-section",
+  DREAM_ROLE: "dream-role-section",
+  GOALS_WORK_STYLE: "goals-work-style-section",
+};
+
+const EmployeeProfile_Fragment = graphql(/** GraphQL */ `
+  fragment EmployeeProfile on User {
+    isGovEmployee
+    workEmail
+    isWorkEmailVerified
+  }
+`);
+
+interface EmployeeProfileProps {
+  userQuery: FragmentType<typeof EmployeeProfile_Fragment>;
+}
+
+const EmployeeProfile = ({ userQuery }: EmployeeProfileProps) => {
+  const intl = useIntl();
+  const paths = useRoutes();
+  const user = getFragment(EmployeeProfile_Fragment, userQuery);
+  const { isGovEmployee, workEmail, isWorkEmailVerified } = user;
+
+  if (
+    !isVerifiedGovEmployee({ isGovEmployee, workEmail, isWorkEmailVerified })
+  ) {
+    throw new UnauthorizedError(
+      intl.formatMessage({
+        defaultMessage: "Not a verified employee",
+        id: "Ljv0T9",
+        description: "Error message for unauthorized employee access",
+      }),
+    );
+  }
+
+  const pageTitle = intl.formatMessage({
+    defaultMessage: "Your empoloyee profile",
+    id: "HwW7BZ",
+    description: "Page title for a users employee profile",
+  });
```

Comment:
Your employee profile*

Reply:
Good eye, fixed in [update copy](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12371/commits/2af1dcd7dc2538b005490a953a68032cfe98456a)